models.py

```diff
@@ -0,0 +1,48 @@
+import requests
+
+
+def get_bitstreams(item, file_type):
+    """"Retrieves the bitstreams of the specified item."""
+    elements = item.iter()
+    link_tag = '{http://www.w3.org/2005/Atom}link'
+    for element in [e for e in elements if e.tag == link_tag
+                    and 'type' in e.attrib.keys()
+                    and e.attrib['type'] == file_type]:
+        link = element.attrib['href']
+        yield link
+
+
+def extract_handle(item):
+    """"Retrieves the bitstreams of the specified item."""
+    elements = item.iter()
+    handle = ''
+    link_tag = '{http://www.w3.org/2005/Atom}link'
+    for element in [e for e in elements if e.tag == link_tag
+                    and e.attrib['rel'] == 'alternate']:
+        handle = element.attrib['href']
```

Comment:
@gravesm I don't believe so, there's no reason why there should be two handles. Should I be using a similar method as with the title field below?

Reply:
Unfortunately, you can't quite get away with findtext in this instance. I haven't worked much with elementtree, but I was curious how much XPath it supports, since I know it's a limited subset. If you expect to always be able to find a handle, you could do:
```python
item.find('atom:link[@rel="alternate"]', namespaces=NS).get('href')
```
If there isn't a handle, this will throw an exception because `find` returns `None`. This may or may not be a good thing depending on what your requirements are.

Reply:
Actually, I just remembered that there are deleted items captured in the OAI file which wouldn't have handles and throw exceptions. So I don't think that this would work for this situation

Reply:
Made a few tweaks to get this to work